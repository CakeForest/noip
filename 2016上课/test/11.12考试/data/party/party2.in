8 10
2 1
3 2
4 3
5 4
6 5
7 6
8 7
1 2 7
2 1 5
3 1 5
4 1 9
5 2 2
6 1 10
7 2 9
8 1 8
3 2 10
7 2 2
10
7
1
5
3
2
7
6
7
8
8
