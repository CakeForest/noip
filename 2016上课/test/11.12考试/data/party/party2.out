14
0
12
10
5
14
22
14
22
22
